irods/test/scripts/test003.bats

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bats
+
+. "$BATS_TEST_DIRNAME"/funcs
+PYTHON=python3
+
+# Setup/prerequisites are same as for login_auth_test.
+# Run as ubuntu user with sudo; python_irodsclient must be installed (in either ~/.local or a virtualenv)
+#
+
+setup()
+{
+ :
+}
+
+teardown()
+{
+ :
+}
```

Comment:
Is this test file a WIP?

It's not clear to me whether it's complete yet.

Reply:
probably best for the moment to consider it a WIP.  Will review again myself this afternoon....

Reply:
Okay. If it's not absolutely necessary for this PR, let's punt it and get the primary work completed.

As long as there's issues linked to things, we can always handle them in PRC 3.

Reply:
Tests needed , so I'll include them and will head each with a comment as to what it is testing.

Reply:
I believe these test scripts are cleaned up, but we'll have a final review tomorrow probably, so will leave this unresolved.

Reply:
Tests seem good now.